### DISCUSSION TITLE
本人无时间维护新功能，仅做基本维护，建议移步新分支TKaxv-7S

### DISCUSSION THREAD

**Discussion by constanline** [constanline/XQuickEnergy#548] [Announcements/公告]
今天朋友给推了这个新分支。看了下，作者应该是做开发的，逻辑性不错，有抽象性化，而且愿意花时间重构屎山代码。感动！
主分支：https://github.com/TKaxv-7S/XQuickEnergy/
tk分支：https://github.com/TKaxv-7S/XQuickEnergy/tree/main_tk

**Comment by TKaxv-7S** [comment]
非常感谢你为这个项目的付出！你维护期间增加了非常多的功能，才有现在这么完善的芝麻粒。也感谢其他为这个项目付出的朋友，我会和大家一起努力让这个项目继续保持活力！

**Reply by lee1ei** [reply]
谢谢，各位辛苦了

**Reply by chengnan049** [reply]
大佬们都辛苦了！

**Reply by haffers** [reply]
大佬们辛苦了

**Comment by ritaferni** [comment]
感谢大神们的辛苦付出！！

**Comment by komican** [comment]
github.com/TKaxv-7S/Sesame-TK 仓库已存档了，@LazyImmortal 维护的 Sesame 不存在页面，在哪里可以找到？

**Reply by kimshift** [reply]
后面不开源了所以找不到

**Reply by b3b41020** [reply]
请问如何激活
